src/main/webapp/styles.js

```diff
@@ -0,0 +1,44 @@
+const propertiesReader = require("properties-reader");
+const fs = require("fs");
+
+const defaults = {
+  "primary-color": "#1890ff",
+  "info-color": "#1890ff",
+  "link-color": "#1890ff",
+  "font-size-base": "2px",
+  "border-radius-base": "2px",
+};
+const iridaConfig = "/etc/irida/irida.conf";
+const propertiesConfig = "../resources/ca/corefacility/bioinformatics/irida/config/styles.properties";
+
+function formatAntStyles() {
+  const colourProperties = {};
+  const re = /styles.ant.([\w+-]*)/;
+
+  try {
+    if (fs.existsSync(propertiesConfig)) {
```

Comment:
The user can set the properties in irida.conf or styles.properties correct? Do we want to allow the user to mix and put some styles in both files? I think this `if` and the `if` blocl below should be changed to an if/else block

Reply:
Updated in 264b0e5cc9ce579cf439f25630edddc6cf4595f5